zcash_primitives/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this library adheres to Rust's notion of
 
 ## [Unreleased]
 
+### Added
+- `zcash_primitives::transaction`
```

Comment:
I have not checked this for completeness; we should do a wholesale changelog review before release.